src/Tasks/Common/Resources/xlf/Strings.zh-Hant.xlf

```diff
@@ -426,6 +426,11 @@ The following are names of parameters or literal values and should not be transl
         <target state="translated">NETSDK1096: 最佳化組件的效能失敗。您可以排除失敗的組件不予最佳化，或將 ReadyToRun 屬性設定為 false。</target>
         <note>{StrBegin="NETSDK1096: "}</note>
       </trans-unit>
+      <trans-unit id="ReadyToRunCompilationHasWarnings">
```

Comment:
I expect a test to pass and force you to name this with _Info suffix. This is so we can enforce error codes.

Reply:
I can add an _Info suffix